bip-0068.mediawiki

```diff
@@ -0,0 +1,199 @@
+<pre>
+  BIP: 68
+  Title: Consensus-enforced transaction replacement signalled via sequence numbers
+  Author: Mark Friedenbach <mark@friedenbach.org>
+  Status: Draft
+  Type: Standards Track
+  Created: 28-05-2015
```

Comment:
I think we switched all dates in the BIPs repository to ISO8601 in d83755efa4207, so 2015-05-28 would probably be better.